extensions/arc/deployment/src/main/java/io/quarkus/arc/deployment/IgnoreSplitPackageBuildItem.java

```diff
@@ -0,0 +1,30 @@
+package io.quarkus.arc.deployment;
+
+import java.util.Collection;
+
+import io.quarkus.builder.item.MultiBuildItem;
+
+/**
+ * Allows extensions to programmatically exclude certain packages from split package detection which is executed by
+ * {@link SplitPackageProcessor}. Extensions are encouraged to solve split package issues and this build item should
+ * be used primarily as temporary workaround.
+ * <p>
+ * A package string representation can be:
+ * <ul>
+ * <li>a full name of the package, i.e. {@code org.acme.foo}</li>
+ * <li>a package name with suffix {@code .*}, i.e. {@code org.acme.*}, which matches a package that starts with provided
+ * value</li>
+ *
+ */
+public class IgnoreSplitPackageBuildItem extends MultiBuildItem {
```

Comment:
AFAIK a build item must be `final`.

Reply:
Ah, very good catch! I always miss these...

Reply:
Fixed

Reply:
ArchUnit _cough_ 😉

Reply:
I had a PR around something similar...